frontend/src/components/ReferralCreationModal/ReferralCreationModal.tsx

```diff
@@ -17,34 +19,46 @@ const style = {
 }
 
 export default function ReferralCreationModal(props: ReferralCreationModalProps & Omit<ModalProps, 'children'>) {
-  const { jobPost, ...modalProps } = props
-  const [activeStep, setActiveStep] = useState(0)
+  const { jobPost, closeModal, ...modalProps } = props
+  const [ activeStep, setActiveStep ] = useState(0)
+
+  // Array of [label, component] pairs
+  const steps: [string, ReactElement][] = [
+    // TODO
+    ['Personal', <div/>],
+    // TODO
+    ['Recommendation', <div/>],
+    // TODO
+    ['Documents', <div/>],
+    // TODO
+    ['Review', <ReviewPage/>],
+  ]
```

Comment:
Figured something like this would make things a little cleaner.. if anyone has a better alternative we can definitely change it.